src/ui/mouse.cpp

```diff
@@ -1315,7 +1315,7 @@ static int SendResource(const Vec2i &pos)
 					if (unit.Type->ResInfo[res]
 						&& unit.Type->ResInfo[res]->TerrainHarvester
 						&& mf.playerInfo.IsExplored(*unit.Player)
-						&& mf.IsTerrainResourceOnMap(res)
+					  /*&& mf.IsTerrainResourceOnMap(res)*/
```

Comment:
A code comment here might be a good idea

Reply:
Fixed